crypto/s2n_pkey.c

```diff
@@ -33,20 +38,16 @@ int s2n_pkey_zero_init(struct s2n_pkey *pkey)
     pkey->verify = NULL;
     pkey->encrypt = NULL;
     pkey->decrypt = NULL;
-    pkey->free = NULL;
```

Comment:
You probably explained this in a previous PR that I wasn't a reviewer for, but why are we able to get rid of those two functions?

Reply:
They're a no-op for the evp implementation, so I don't see a reason to keep them around. We can always add them back if we need them for a later implementation.